src/nativeMain/kotlin/util/Util.kt

```diff
@@ -10,6 +10,8 @@ import xlib.*
 
 fun getConfigDir(): String? = getenv("XDG_CONFIG_HOME")?.toKString() ?: getenv("HOME")?.toKString()?.plus("/.config")
 fun Boolean.toInt(): Int = if (this) 1 else 0
+var windowCursors: MutableMap<Window, Cursor> = mutableMapOf()
+var windowCursorInitialized: MutableMap<Window, Boolean> = mutableMapOf()
```

Comment:
Are these for using later?

Reply:
windowCursorInitialized remains to stop the WM from continually trying to recreate the cursor if it's used in the wrong place, but windowCursors can be removed